<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel{
color:rgb(106, 106, 191);
font: 11pt &quot;MS Shell Dlg 2&quot;;
}

QRadioButton{
color:rgb(106, 106, 191);
font: 9pt &quot;MS Shell Dlg 2&quot;;
}

QFrame{
background-color:white;}


QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
font-size: 11px;
padding-left: 5px;
padding-right: 5px;
min-width: 110px;
max-width: 110px;
min-height: 14px;
max-height: 14px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Light Detector</string>
    </property>
   </widget>
   <widget class="QFrame" name="toggle_btm_showMask">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1131</width>
      <height>671</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btm_Run">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Contour Detection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_restart">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Restart</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_ShowLV">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Lightvectors</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_image">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Image</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_image">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>851</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_mask">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>610</y>
       <width>800</width>
       <height>500</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_selection">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Part of Contour</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_saveSelection">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Selection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_deleteSelection">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete Selection</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_intensity">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate Intensity</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btm_ShowN">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>122</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Show Normals</string>
     </property>
    </widget>
   </widget>
   <zorder>toggle_btm_showMask</zorder>
   <zorder>label_1</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
